frontend/common/stores/feature-list-store.js

```diff
@@ -414,7 +414,25 @@ const controller = {
   ) => {
     store.saving()
     API.trackEvent(Constants.events.EDIT_FEATURE)
-
+    const env = ProjectStore.getEnvironment(environmentId)
+    let environment_feature_versions = null
```

Comment:
The API seemingly doesn't like explicitly setting `null` here so this is breaking the existing change request workflow. I can update the API, but the quickest fix would probably be the following (or not sending the key at all). 

```suggestion
    let environment_feature_versions = []
```

Reply:
Yep ok makes sense